sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldPredictionAccuracy.cs

```diff
@@ -5,9 +5,9 @@
 
 namespace Azure.AI.FormRecognizer.Custom
 {
-    public class LabeledFieldAccuracy
+    public class FieldPredictionAccuracy
     {
-        internal LabeledFieldAccuracy(FormFieldsReport_internal field)
+        internal FieldPredictionAccuracy(FormFieldsReport_internal field)
```

Comment:
I think it might be pretty easy to map this type to `FormFieldsReport` and rename the `Label` property. You'll get one less thing to wrap.